server/tests/trip.test.js

```diff
@@ -190,7 +191,7 @@ describe('Trip test suite', () => {
         done(err);
       });
   });
-  it('12.should get all trip created by the users he/she manage', (done) => {
+  it('12.should not get all trip created by the users he/she manages', (done) => {
```

Comment:
Remove the number and make the description more meaningful.

Reply:
removed